src/templates/legal/regulation.html.tt

```diff
@@ -0,0 +1,198 @@
+<div class="static_full">
+    <h1 class="center-text">[% l("Regulatory Information") %]</h1>
+    <div class="gr-parent gr-padding-30">
+        <p class="center-text">
+            [% l("The Binary.com service (formerly known as BetOnMarkets.com) is owned and operated by Binary Ltd. Fully licensed and regulated, Binary.com has been providing financial trading to a worldwide audience of over 1,000,000 accounts since 1999.") %]
```

Comment:
remove "(formerly known as BetOnMarkets.com)". It's too many years ago, we don't need to mention it any more.